doc/dnsseed-policy.md

```diff
@@ -1,54 +1,53 @@
 Expectations for DNS Seed operators
 ====================================
 
-Zcash attempts to minimize the level of trust in DNS seeds,
-but DNS seeds still pose a small amount of risk for the network.
-As such, DNS seeds must be run by entities which have some minimum
-level of trust within the Zcash community.
+Zcash attempts to minimize the level of trust in DNS seeds, but DNS seeds still
```

Comment:
This is a document from upstream, so I would prefer we don't reformat it in this way (for the same reason we don't reformat the entire codebase), unless doing so is required for correct book rendering.

Reply:
Undid this change.